lib/ramble/ramble/test/cmd/config.py

```diff
@@ -148,43 +148,45 @@ def test_merged_env_vars_section(mock_low_high_config):
     )
 
 
-def test_merged_spack_section(mock_low_high_config):
+# DEPRECATED: Remove `spack` when removed
```

Comment:
No because the deprecation warning isn't on the schema itself, it's on the usage of the schema (so this test wouldn't fail if spack wasn't removed when the version was updated)